test/catalog/constraints_tests_util.cpp

```diff
@@ -102,6 +102,17 @@ catalog::Column ConstraintsTestsUtil::GetColumnInfo(int index) {
       return column;
     } break;
 
+    case 114: {
+      auto column = catalog::Column(type::Type::INTEGER,
+                                    25,  // Column length.
```

Comment:
Could you add more comments for "25"?  "Column length"  doesn't really help me understand the code.